services/src/unit-test/java/uk/gov/moj/sdt/services/BulkSubmissionServiceTest.java

```diff
@@ -188,11 +166,122 @@ void testSaveBulkSubmission() throws IOException {
         bulkSubmissionService.saveBulkSubmission(bulkSubmission);
 
         // Verify the Mock
-        verify(mockGenericDao, times(1)).persist(bulkSubmission);
+        verify(mockGenericDao).persist(bulkSubmission);
+
+        assertTrue(true, EXPECTED_TO_PASS);
+    }
+
+    @Test
+    void testSaveBulkSubmissionWithDebug() throws IOException {
```

Comment:
Perhaps rename this method as testSaveBulkSubmissionTargetAppNotNull().  This test is only checking one of the debug messages and that's only to confirm the path taken when target app is not null.

Reply:
Done